R/aggregateData.R

```diff
@@ -22,6 +22,7 @@ aggregateData <- function (dat, subunits, units, aggregatemissings = NULL, renam
                         recodedData = TRUE, suppressErr = FALSE, recodeErr = "mci", verbose = FALSE) {
```

Comment:
line 21: should I test subunits and units for something?

Reply:
yes, please. see prep2GADS function in GADS output branch